MotorizationRate.ipynb

```diff
@@ -148,7 +148,7 @@
     }
```

Comment:
<p>define FUNCTIONS to make things:</p><p> </p><pre class="ql-syntax" spellcheck="false">def select_categorical_columns(df, numerical_columns=num_cols):
    return [col for col in df.columns if col not in numerical_columns]
</pre><p> </p><p>this is a very good habit to have.</p><p> </p><ul><li>It prevents from having intermediary variables in global scope (hence reduces bugs).</li><li>It allows for easier testing/debugging.</li><li>Reusable pieces</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Adanel/MotorizationRate/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>